### PR TITLE
Add MIT license to footer, contributor guidelines, and LICENSE file

### PR DESCRIPTION
**Pull request by rachelwhitton**
@davidstrauss can you review and +1? I'll ship ASAP.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,29 @@
+MIT License applies only to code snippets
```
Comment (davidstrauss) [inline LICENSE]: The Creative Commons Attribution-ShareAlike 4.0 International applies to the everything. Code snippits are, additionally, available under the MIT License.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Help us create relevant and useful content for developers like yourself. See som
   Trying to edit or create a file in this repository will create your fork automatically.
 * [Clone your fork locally](https://help.github.com/articles/cloning-a-repository/).
 
-**NOTE**: All contributions will be licensed under  [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) to Pantheon.
+**NOTE**: All contributions will be licensed under  [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) and code snippet contributions will be licensed under [The MIT License](http://opensource.org/licenses/MIT) to Pantheon.
```
Comment (davidstrauss) [inline CONTRIBUTING.md]: ...code snippet contributions will be additionally licensed...

(code snippits are still under CC-BY-SA 4.0)

File: source/_partials/footer_content.html [hunk 3]
```diff
@@ -5,4 +5,5 @@
 <div class="pio-docs-footer">
 <div id="copyright">&copy; {{ "now"|date("Y") }}
   <a href="https://pantheon.io/" target="blank"> Pantheon</a>  &middot; 717 California Street, San Francisco, CA <br />
-  <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a>   This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.</div>
+  <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/" target="blank"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a>   All work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/" target="blank">Creative Commons Attribution-ShareAlike 4.0 International License</a>.<br />
+  Code snippetes are licensed under <a href="http://opensource.org/licenses/MIT" target="blank">The MIT License</a> </div>
```
Comment (davidstrauss) [inline source/_partials/footer_content.html]: snippits

**Comment by davidstrauss** [comment]
LGTM!

**Comment by rachelwhitton** [comment]
AWESOME! @davidstrauss thank you so much for your help! I'll merge now

**Pull request merged**